docs/sidebars.js

```diff
@@ -143,6 +143,11 @@ const sidebars = {
           label: 'Upgrade your cluster',
           id: 'workflows/upgrade',
         },
+        {
+          type: 'doc',
+          label: 'Deploy an application',
```

Comment:
I'd align this with the heading of the actual page and change this to `Expose a service`.